binstar_client/utils/projects/__init__.py

```diff
@@ -0,0 +1,102 @@
+import inspect
+import logging
+import os
+from os.path import basename, isdir, join, relpath
+from binstar_client.errors import BinstarError
+from .filters import filters
```

Comment:
there is a 100% chance that we will extract this into another library. I'll leave this the way it is